content/docs/userspace/dist/guide.md

```diff
@@ -0,0 +1,281 @@
+    glob-ames+[~zod 0v0]
+    version+[0 0 1]
+    website+'https://urbit.org/docs/userspace/dist/guide'
+    license+'MIT'
+==
+```
+
+You can refer to the [Docket File](/docs/userspace/dist/docket) documentation for more details of what is required. In brief, the `desk.docket-0` file contains a `hoon` list of [clauses](/docs/userspace/dist/docket) which configure the appearance of the app tile, the source of the [glob](/docs/userspace/dist/glob), and some other metadata.
+
+We've given the app a [`%title`](/docs/userspace/dist/docket#title) of "Hello", which will be displayed on the app tile and will be the name of the app when others browse to install it. We've given the app tile a [`%color`](/docs/userspace/dist/docket#color) of `#8188C9`, and also specified the URL of an [`%image`](/docs/userspace/dist/docket#image) to display on the tile.
+
+The [`%base`](/docs/userspace/dist/docket#base) clause specifies the base URL path for the app. We've specified "hello" so it'll be `http://localhost:8080/apps/hello/...` in the browser. For the [glob](/docs/userspace/dist/glob), we've used a clause of [`%glob-ames`](/docs/userspace/dist/docket#glob-ames), which means the glob will be served from a ship over Ames, as opposed to being served over HTTP with a [`%glob-http`](/docs/userspace/dist/docket#glob-http) clause or having an Eyre binding with a [`%site`](/docs/userspace/dist/docket#site) clause. You can refer to the [glob](/docs/userspace/dist/glob) documentation for more details of the glob options. In our case we've specified `[~zod 0v0]`. Since `~zod` is the fakeship we'll install it on, the `%docket` agent will await a separate upload of the `glob`, so we can just specify `0v0` here as it'll get overwritten later.
+
+The [`%version`](/docs/userspace/dist/docket#version) clause specifies the version as a triple of major version, minor version and patch version. The rest is just some additional informative metadata which will be displayed in _App Info_.
+
+So let's save that to the `desk.docket-0` file and have a look at our desk:
+
+```
+[user@host hello]$ ls
+desk.docket-0  desk.ship  lib  mar  sur  sys.kelvin
+```
+
+That's everything we need for now.
+
+## Install
+
+Let's spin up a fakezod in which we can install our desk:
+
+```
+[user@host hello]$ cd ~/piers/fake
+[user@host fake]$ urbit -F zod
+```
+
+Once our fakezod is booted, we'll need to create a new `%hello` desk for our app and mount it. We can do this in the dojo like so:
+
+```
+> |merge %hello our %base
+>=
+> |mount %hello
+>=
+```
+
+Now, back in the Unix terminal, we should see the new desk mounted:
+
+```
+[user@host fake]$ cd zod
+[user@host zod]$ ls
+hello
+```
+
+Currently it's just a clone of the `%base` desk, so let's just delete it entirely:
+
+```
+[user@host zod]$ rm -r hello
+```
+
+Next, we'll copy in the `hello` desk we created earlier. We must use `cp -LR` to resolve all the symlinks:
+
+```
+[user@host zod]$ cp -LR ~/urbit-git/pkg/hello hello
+```
```

Comment:
Doesn't the `rm` nuke the _entire_ directory? Does the `cp` recreate it? If not, maybe `rm -r hello/*` instead. That also makes it more obvious we want to remove its _contents_.